intra_hutch_main.py

```diff
@@ -39,16 +50,16 @@
 
 subprocess.run(["mkdir", "-p", pv_lists_path])
 
-total_iterations = int(input("Enter total iterations of random sampling: "))
-interval = float(input("Enter max interval of each iteration in seconds: "))
+# total_iterations = int(input("Enter total iterations of random sampling: "))
+# interval = float(input("Enter max interval of each iteration in seconds: "))
 
 
 
 
-# Run through all the IOCs of a hutch and configure the JSON file.
+### Run through all the IOCs of a hutch and configure the JSON file.
 with open(ioc_path, 'r') as readtxt:
     ioc_list = [line.strip() for line in readtxt.readlines()]
-    print("Estimated wait time: " + str(int(len(ioc_list) * total_iterations * (interval/2) / 60 + 2)) + " minutes.")
+    print("Estimated wait time: " + str(int(len(ioc_list) * args.total_iterations * (args.interval/2) / 60 + 2)) + " minutes.")

```

Comment:
Ideally a line like this would have you calculate the value first, and then on the next line generate your print statement.